### PR TITLE
fix: Favorite chips margins

### PR DESCRIPTION
**Pull request by KristianSelnas**
Fixes an issue found during testing of https://github.com/AtB-AS/kundevendt/issues/8088

See https://github.com/AtB-AS/kundevendt/issues/8088#issuecomment-1710018616

### CODE REVIEW

**Comment by rosvik** [comment]
Screenshot 2023-09-12 at 15 04 20

Looks a lot better, but still doesn't line up on the right edge when scrolling all the way to the right. Should be 12px off the screen edge I think.

**Comment by KristianSelnas** [comment]
> Screenshot 2023-09-12 at 15 04 20
> 
> Looks a lot better, but still doesn't line up on the right edge when scrolling all the way to the right. Should be 12px off the screen edge I think.

Strange. I cannot figure out why, and I don't see any immediate fixes. There is only 12px (`theme.spacing.medium`) padding on the right edge. I have no idea why it appears to be more. Personally, I don't think it's a big deal now. It is a whole lot better to get this fix in before the release with this minor detail, than not having it this fix at all.

**Comment by gorandalum** [comment]
> Strange. I cannot figure out why, and I don't see any immediate fixes. There is only 12px (`theme.spacing.medium`) padding on the right edge. I have no idea why it appears to be more. Personally, I don't think it's a big deal now. It is a whole lot better to get this fix in before the release with this minor detail, than not having it this fix at all.

It's the `marginRight` stuff in `FavoriteChips` that is acting up. The add button should have no margin right. But it was here before, so no blocker.

**Review by gorandalum — APPROVED** [review]

**Comment by rosvik** [comment]
simulator_screenshot_1DF1B26D-D4A4-4857-BD57-8D6A6006B186

Looks like it's happening because all buttons have `small` margin on the right, so what we're seeing is 8+12px padding on the screen edge. I think there has been something like `marginRight: medium - small` before.

But you're right, this is a lot better 👍

File: src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/Dashboard_RootScreen.tsx [hunk 1]
```diff
@@ -435,6 +435,7 @@ const useStyle = StyleSheet.createThemeHook((theme) => ({
   },
   favoriteChips: {
     marginTop: theme.spacings.medium,
+    paddingHorizontal: theme.spacings.medium,
```
Comment (rosvik) [inline src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/Dashboard_RootScreen.tsx:438]: This seems to line up for me. Not the most beautiful code, but more beautiful app 😄 

```suggestion
    paddingLeft: theme.spacings.medium,
    paddingRight: theme.spacings.medium - theme.spacings.small,
```

**Pull request merged**